src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,59 @@
+# approved scripts
+metadataGatheringDisabled=Metadata gathering disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedExpandCode=Code
+approvedLanguage=Language
+usageCount=# of uses
+usageCount_tooltip=Uses since the introduction of metadata
+lastTimeUsed=Date of last use
+wasPreapproved_header_tooltip=The presence of a green tick means the script was approved during the configuration. \
+  Otherwise it was created by a user without the permission to run scripts and was then approved using this page.
+lastApprovalTime=Date of the last approval
+lastApprovalTime_tooltip=A single script can be approved multiple times. It occurs every time the related configuration is saved.
+lastKnownApproverLogin=Last approver
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. \
+  It could be because the request was created before the metadata was introduced or the code calling this script did not provide user information.
+noRequestApprovalTimeSinceMetadata=N/A
+noRequestApprovalTimeSinceMetadata_tooltip=It was requested before the introduction of metadata.
+noContextItemSinceMetadata=N/A
+noContextItemSinceMetadata_tooltip=The context item was not provided by the code asking for approval.
+
+emptyMetadata=There is no metadata for this approval
+emptyMetadata_tooltip=If the script is used after the metadata introduction, some metadata will be displayed here. \
```

Comment:
This tooltip lacks discoverability.

Reply:
I tried with a help icon but it will require to have newer version of core to have the latest help icon, so keeping it simple for the moment with `cursor: help`

<details>

<summary>Cursor</summary>

![Screenshot_2020-09-02_084251_001](https://user-images.githubusercontent.com/2662497/91940923-955a7780-ecf8-11ea-8c60-4c81735f8788.png)


</details>